manifests/check.pp

```diff
@@ -107,11 +107,18 @@
 #   Set this to 'absent' to remove it completely.
 #   Default: undef
 #
+# [*content*]
```

Comment:
They're slightly different.  Custom is a hash which gets merged into the scope of the check configuration at checks.mycheck.  Content is a hash merged onto the top level JSON object.  Without content, there isn't a way to specify configuration for the [sensu-mailer-plugin](https://github.com/sensu-plugins/sensu-plugins-mailer) example.

I kept this in the define, please take a look at the Puppet DSL behavior replacing the parser function in the latest patch.